br/pkg/lightning/config/config.go

```diff
@@ -70,7 +70,10 @@ const (
 	// ErrorOnDup indicates using INSERT INTO to insert data, which would violate PK or UNIQUE constraint
 	ErrorOnDup = "error"
 
-	KVWriteBatchSize        = 32768
+	KVWriteBatchCount = 32768
+	// KVWriteBatchSize batch size when write to TiKV.
+	// this is the default value of linux send buffer size(net.ipv4.tcp_wmem) too.
+	KVWriteBatchSize        = 16 * units.KiB
```

Comment:
It's too small considering a large row KV. Maybe we should use 4M which is the maximun value of it?

Reply:
it depends on how often do customer tune network parameters, if user do set to 4M
`(256 * (accumulate 16k + serialize + append to linux send buffer)) + send to network` compared to `(accumulate 4M + serialize + append to linux send buffer) + send to network`, seems no much difference, will test it using qa env

Reply:
i misunderstand, it works like this:
```
When a new TCP connection is established, a Send Buffer will be created using the default value (16KB); the buffer size will then be automatically adjusted within the maximum and minimum boundaries as needed and based on usage.
```